glyphs-reader/src/plist.rs

```diff
@@ -472,6 +505,289 @@ impl From<BTreeMap<String, Plist>> for Plist {
     }
 }
 
+pub(crate) fn parse_int(s: &str) -> Result<i64, Error> {
+    if numeric_ok(s) {
+        if let Ok(num) = s.parse::<i64>() {
+            return Ok(num);
+        }
+        if let Ok(num) = s.parse::<f64>() {
+            return Ok(num as i64);
+        }
+    }
+    Err(Error::ExpectedNumber)
+}
+
+pub(crate) fn parse_float(s: &str) -> Result<f64, Error> {
+    if numeric_ok(s) {
+        if let Ok(num) = s.parse::<f64>() {
+            return Ok(num);
+        }
+    }
+    Err(Error::ExpectedNumber)
+}
+
+/// This type can be parsed from a Plist string
+pub trait FromPlist
+where
+    Self: Sized,
+{
+    fn parse(tokenizer: &mut Tokenizer<'_>) -> Result<Self, Error>;
```

Comment:
you can actually completely omit the `<'_>` here, which would save a little bit of noise.

Reply:
Done